src/markdown-pages/automate-workflows/error-inbox/set-up-error-inbox.mdx

```diff
@@ -0,0 +1,199 @@
+![channel wrong state error](../../../images/error-inbox/channel-wrong-state.png)
+
+You see two options in the dropdown here. Select **Resolve**.
+![set error status to resolve](../../../images/error-inbox/resolve-error.png)
+
+Once the status is updated, you no longer see the error on the main screen. However, if the error still persists, then it'll automatically be marked as unresolved.
+
+### Set the error status as 'Ignored'
+Some errors are expected and part of the system as it operates. Set those errors as ignored:
+
+Navigate to **Errors inbox** in New Relic One.
+![errors inbox](../../../images/error-inbox/error-inbox.png)
+
+Take a quick scan of the errors here. There's one **builtins:AssertionError**.
+![assertion-error](../../../images/error-inbox/builtin-assertion-error.png)
+
+This error is expected since the payment was aborted by the user and there's nothing you can do about it. Ignore this error:
+
+Click **Unresolved** against the error and then select **Ignore** from the dropdown.
+![ignore error](../../../images/error-inbox/ignore-assertion-error.png)
+
+Once the status is updated, you no longer see the error on the main screen until you stop ignoring or change the filter to include ignored error groups. 
+
+## Change filter
+By default, errors inbox show all the unresolved errors on the main screen. Once the error status is updated to resolved or ignored, you no longer see the error on the main screen. However, you can change the filter to view these errors.
+
+The main screen of errors inbox has a filter pane right below the top nav.
+![filters pane of errors inbox](../../../images/error-inbox/filters-pane.png)
+
+Click **Unresolved** to change the filter value.
+![chnage filter value](../../../images/error-inbox/change-filter.png)
+
+Here, you see three options in the dropdown. Select **Ignored**.
+![view ignored error groups](../../../images/error-inbox/view-ignored-errors.png)
+
+Your errors inbox now show all the ignored error groups.
+![ignored error groups](../../../images/error-inbox/ignored-error-groups.png)
+
+## Assign errors
+Change the errors inbox filter to show all the unresolved errors.
+![unresolved error groups](../../../images/error-inbox/unresolved-errors.png)
+
+Oops! you were told that somebody fixed **pika.exceptions:ChannelWrongStateError** error but you still see it under unresolved errors. You don't know who fixed this error previously and there's no way for you to find that out. To avoid such situation in future, assign the error from inside errors inbox.
+
+Click the human icon on the far right side of the error group.
+![assign error](../../../images/error-inbox/assign-error.png)
+
+Enter the email address of the person you want to assign error to, and click **+**. 
+![assign error via email](../../../images/error-inbox/assign-error-via-email.png)
+
+The error is now assigned and its now easier for you to keep a track of who worked on the error.
+
+<Callout variant='important'>
```

Comment:
I feel like this callout should be kept as important.

Reply:
The way I think of these is:

- tip: Something that's nice for the user to know, but not necessary for them to read to complete the lab.
- important: Something that the user really should read, but isn't absolutely critical
- caution: Something that's critical that the user read

In this case, I think this is a tip because it's letting the reader know about a feature that doesn't exist but will in the future. Feel free to dispute these definitions, and we can chat about that, but I hope that helps.